### DISCUSSION TITLE
Check component readonly not recognized?

### DISCUSSION THREAD

**Discussion by svenbaus** [Megabit/Blazorise#4079] [Q&A] [answered]
Hello,

I'm using the <Check> Component and would like to have it only readonly, so I use this code:

`<Check TValue="Boolean" Checked="@processStep.RunDirectly" ReadOnly/>`

But in GUI, the user can change the check value, is there a bug?

**Comment by stsrki** [comment, marked as answer]
It seems it is a problem with the HTML checkbox itself. It cannot be readonly. https://stackoverflow.com/questions/155291/can-html-checkboxes-be-set-to-readonly

We will probably need to find some workaround to make it read-only.

**Reply by svenbaus** [reply]
Thanks for the fast reply :). I'll switch to the "Switch" component.
